versions-maven-plugin/src/main/java/org/codehaus/mojo/versions/AbstractVersionsReport.java

```diff
@@ -209,6 +209,12 @@ protected void executeReport(Locale locale) throws MavenReportException {
         }
     }
 
+    /**
+     * Returns the bundle name for the underlying report renderers
+     * @return bundle name for the underlying report renderers
+     */
+    protected abstract String getBundleName();
+
```

Comment:
in `org.apache.maven.reporting.MavenReport` there is replaced `getOutputName` by `getOutputPath` - maybe we also use `getOutputPath` here ...

Reply:
I don't remember the reason why I did that. I'll see if we can use `getOutputPath`.